CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## v1.4.0 [unreleased]
 
+### Configuration Changes
+
+#### `[collectd]` Section
+
+* `parse-multivalue-plugin` was added with a default of `split`.  When set to `split`, multivalue plugin data will be split into separate measurements.  When set to `join`, multivalue plugin will be stored as a single multi-value measurement.  
```

Comment:
This config option needs to be added to the sample config in `etc/config.sample.toml` too.